### DISCUSSION TITLE
GitHub Education Fails for False Reasons

### DISCUSSION THREAD

**Discussion by lara-rium** [community/community#58241] [GitHub Education]
### Hi everyone,

It said my school’s name isn’t written in the documents I uploaded, even thought the name and the logo of my university is there.

It also says my profile doesn’t have the name in the documents I provided, even though I’ve explained my situation. It’s my legal name and I don’t use it in real life. The surname is the same.

**Comment by the-officialjosh** [comment]
I understand that you may be frustrated by the verification process, but it's important to note that measures like these are in place to ensure compliance with the guidelines set by GitHub. One of the reasons for verifying school enrollment is to prevent misuse of student email addresses, such as siblings or relatives using them to access student benefits.

That being said, it can still be frustrating if you're facing difficulties even though you have provided the necessary documents and information. In such cases, it's best to directly communicate your situation to GitHub's support team. They will be able to review your case more thoroughly and assist you with the verification process.

**Reply by lara-rium** [reply]
Thank you for your answer! I checked [this website](https://support.github.com/contact/education?tags=hubberfy_education) but it says manual verification isn't possible anymore, and the button is disabled saying _Cannot open ticket for this request_. I've still explained my situation per the reasons given to me in my email. 3 of them are invalid and one is explained in my application. I'll put it below:

Hello! I've tried every form of verification I could to apply but it's always rejected. I used my school email, applied in campus, provided a photo of my student ID and images of my student information and my academic transcription. The system noted these issues:
- You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
  - I have explained in the submission that the name in the ID is my legal name and I don't use it in real life. It's my deadname. I believe I have enough of a bio, I even have a profile README
- Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?
  - This sounds like a paraphrasing of the reason above.
- The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible. For example, if your original document is not in English then you might take a photo of the original next to an English translation.
  - I have included my student ID which has my school name and logo. The other documents also have the website they're taken from which uses the metu.edu.tr domain which is my school's domain
- Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
  - The student ID was taken using my camera. The reason I've added the others is that it doesn't show a validity date or current semester.

I've tried time and time again, I hope these errors can be resolved. I'm willing to comply with other methods of verification.

**Reply by the-officialjosh** [reply]
~~Why not try creating a GitHub account with your legal name? This will most likely ensure that even after applying, you will have a higher chance of getting selected. As you are aware, not everyone gets approved for this. It is a better solution if your account is not tied to many repositories.~~

I was about to send that suggestion when I decided to visit your repository. I now understand why you wouldn't have liked to use your given name for GitHub, and creating another account might be too much of a hassle. Resolving this particular situation might be very difficult due to the reasons I explained earlier. The GitHub Education program support operates through ticketing, making it challenging to open this specific ticket elsewhere. I apologize for any inconvenience caused, but this is the extent of my experience in this matter.

**Comment by samus-aran** [comment]
Hi @laralove143 👋 

Thank you for feedback and raising your concerns! We will pass this along to our internal team.

We are extremely proud that the GitHub Global Campus has provided millions of students with access to some of the industry’s most amazing tools and offers all free of charge to compliment their studies. As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. 

However, sometimes we may require a little more evidence from the student applying to help us verify their academic status. This can be due to the system detecting a pattern used by bad actors or simply processing a new type of evidence. While submitting requested information is voluntary, we do thank students for providing this if possible as it speeds up their verification.

While we do take our Community’s feedback seriously we are also mindful to ensure that bad actors impersonating students don't jeopardize our ability to provide academic benefits of value. One aspect of this protection is momentarily compare the evidence provided against your GitHub account information.

I can confirm that once you have submitted your application you can revert this information as it's only required upon each new submission. For more information on how GitHub processes and uses personal data, please see the [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).

Reviewing your rejection reasons here are a few tips:

1. Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process. I would also highlight that School Logos are not detected, if you only have a logo then submit a written translation along with the evidence (in the same photo).
2. If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.

Let us know how you get on 👍

**Reply by lara-rium** [reply]
i understand that you require more data as evidence, but the problem is the presumably automated system gives a lot of false positives and there's no way to contact a real person to explain your situation or a way to request it to be looked over by a human agent

my documents are in english, along with some false positives the issue is that i dont use my legal name on github, or anywhere pretty much, of course i understand this is hard to explain to a bot but a human would understand and maybe ask for additional verification such as biometrics

before the transition to the automated system and lack of support, i had my application approved without further questions, it took a few days but the process this way has taken a month if not more and im still unsure if it'll be verified

**Reply by samus-aran** [reply]
> before the transition to the automated system and lack of support, i had my application approved without further questions, it took a few days but the process this way has taken a month if not more and im still unsure if it'll be verified

We understand your frustration, we do. It's a hard balance to strike keeping bad actors at bay whilst being accommodating to so many different scenarios, institution etc.  In order for GitHub to supply millions of students with Global Campus benefits we had to review at scale. It's not perfect, we know that, but it has enabled so many students to supplement their academic learning. 

> my documents are in english, along with some false positives the issue is that i dont use my legal name on github, or anywhere pretty much, of course i understand this is hard to explain to a bot but a human would understand and maybe ask for additional verification such as biometrics

Let's review these reasons and see if we can identify the blockers. 

First I would ask you to re-apply using the best evidence you have i.e. Student ID card or Academic email. If you are unsuccessful then post the latest rejection reasons here - I'll get a notification.

**Reply by lara-rium** [reply]
hey! thank you for walking me through this, currently im not in campus because of the summer break but i've already done a (rejected) application in campus, do i still have to be in campus to apply again
